package.json

```diff
@@ -38,11 +38,12 @@
   "dependencies": {
     "aguid": "^1.0.4",
     "bignumber.js": "^2.4.0",
-    "canonical-json": "0.0.4",
     "debug": "^2.2.0",
     "eventemitter2": "^2.0.0",
     "ilp-core": "^11.0.0",
```

Comment:
`ilp-core` and `aguid` are no longer a dependencies and `eventemitter2` is only a `devDependency`